libs/components/pages/src/lib/modules/action-hub/types/action-hub-needs-attention.ts

```diff
@@ -16,11 +17,22 @@ export interface SkyActionHubNeedsAttention {
   /**
    * Specifies a link to resolve the action item.
    */
-  permalink: {
+  permalink?: {
     route?: {
       commands: any[];
       extras?: NavigationExtras;
     };
     url?: string;
   };
+  /**
+   * Specifies the modal parameters.
+   */
+  modal?: {
```

Comment:
Do we need the `modal` option if we have a `click` option? I feel like having a specific `modal` option is going to have us chasing modal every time it adds a new feature to maintain parity here. As it is now it doesn't seem to provide the consumer with the modal instance for handling save/cancel/etc.

Reply:
Removed the `modal` option here.